1907-05-27.xml

```diff
@@ -1532,15 +1537,4661 @@
+                        Germany. If these writers had all been silent it is quite possible that
+                        traders and manufacturers in England and in Germany would have been
+                        conscious of a certain rivalry. Mr. Haldane in a spéech the other day, spoke
+                        of “the legitimate antagonism of trade competition.” But it is by no means
+                        clear that this antagonism need have a national or international aspect. Two
+                        spinners in Bolton are much keener rivals one of the other than either of
+                        them of any German competitor,and yet they can manage to live together in
+                        the same town without quarrelling. It would not be easy to find a British
+                        trader or manufacturer who would wish this country to pick a quarrel with
+                        Germany. Some of them would be glad to be helped against German competition
+                        by a protective tariff, but they would not on that account wish the two
+                        nations to enter into the extreme competition of a war. Moreover, men of
+                        business best realise that the best customers are rich industrious nations,
+                        and that Germany and England with or without tariffs on one side or on both
+                        are necessarily bound to have very large commercial dealings with one
+                        another. Apart from manufacture and trade where the two nations are
+                        competitors, but not necessarily unfriendly competitors, what are the
+                        interests of the two nations which clash? We have never been able to
+                        discover them There is no rational cause for a quarrel between England and
+                        Germany, and if a quarrel arises it must be due to some wrong, malice, or
+                        misunderstanding. These things it is not always possible to prevent. They
+                        ought not to arise, and it is the duty of each nation and of each Government
+                        to avoid being the author of any such occasion of dispute, and of course
+                        also to be strong enough in case of a gratuitous quarrel thrust upon it to
+                        defend itself.</p>
+                </div>
+            </div>
             <pb n="3"/>
-            <div type="page" n="3"/>
+            <div type="page" n="3">
+                <div type="masthead">
+                    <head> THE BOYPTIAN Gazerte, MONDAY, MAY 21, 1907.</head>
+                </div>              
+                <cb n="1"/>
+                <div type="section" feature="local">
+                    <head>LOCAL AND GENERAL.</head>
+                   
+                </div>
+                <div type="subsection">
```

Comment:
Instead of using "subsection", try to make small essays in a section as type="item", and put these small items under the sections.